be/src/column/chunk.h

```diff
@@ -251,6 +259,7 @@ class Chunk {
     SlotHashMap _slot_id_to_index;
     TupleHashMap _tuple_id_to_index;
     DelCondSatisfied _delete_state = DEL_NOT_SATISFIED;
+    int64_t _tablet_id;
```

Comment:
```suggestion
    int64_t _tablet_id = 0;
```

Reply:
Should explain why you add a tabletid field in the code.
IMO tablet id is not a good name, for example, some data is not generated from tablet_id, so this field is difficult to understand.